### PR TITLE
using 2d list store layer's _tiles,useful for get tile by coordinate x,y

### PR DESCRIPTION
**Pull request by bung87**
when this get merged , flame also need update `TiledComponent`.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: tiled
-version: 0.2.1
+version: 0.2.2
```
Comment (luanpotter) [inline pubspec.yaml:2]: since this is a breaking change, we should do a bigger release (like 0.3). please revert this change as we can bump the version when we are ready to release all the changes

Comment (luanpotter) [reply pubspec.yaml:2]: but please add your change to the CHANGELOG.md file under `<next>`

Comment (bung87) [reply pubspec.yaml]: ok, done this.
